js/ui/gantt/ui.gantt.size_helper.js

```diff
@@ -6,22 +7,26 @@ export class GanttSizeHelper {
     }
 
     _setTreeListDimension(dimension, value) {
-        this._gantt._$treeListWrapper[dimension](value);
-        this._gantt._ganttTreeList?.setOption(dimension, this._gantt._$treeListWrapper[dimension]());
+        const setter = dimension === 'width' ? setWidth : setHeight;
```

Comment:
Maybe it will be beter to use a map object like this?
```
sizeMethods: {
  get: {
    width: getWidth,
    height: getHeight
  },
  set: {
    width: setWidth,
    height: setHeight
  }
}
```

Reply:
I would prefer to leave refactoring to code owners. 

For example, in current case, defining the `sizeMethods` creates a new member in the `GanttSizeHelper` class. An author may want to keep this member private. Or it may be prefer to declare a helper function (or a map) in the global scope.